lib/ts/supertokens.ts

```diff
@@ -111,13 +114,20 @@ export default class SuperTokens {
                 userMetadataFound = true;
             } else if (recipeModule.getRecipeId() === MultiFactorAuthRecipe.RECIPE_ID) {
                 multiFactorAuthFound = true;
+            } else if (recipeModule.getRecipeId() === TotpRecipe.RECIPE_ID) {
+                totpFound = true;
             }
             return recipeModule;
         });
 
         if (!multitenancyFound) {
             this.recipeModules.push(MultitenancyRecipe.init()(this.appInfo, this.isInServerlessEnv));
         }
+        if (totpFound && !multiFactorAuthFound) {
+            // we want MFA to be enabled if totp is enabled
+            this.recipeModules.push(MultiFactorAuthRecipe.init()(this.appInfo, this.isInServerlessEnv));
+            multiFactorAuthFound = true;
+        }
```

Comment:
Do we actually need/want to do this?

Reply:
Totp cannot function without MFA initialized.